src/Storage/RegressionTests/dataplane_preview.ps1

```diff
@@ -41,6 +41,57 @@ Describe "dataplane test for preview" {
         $Error.Count | should -be 0
 
     }
+
+    It "Cold tier preview" {
+        $Error.Clear()
+
+        $accountName = $testNode.SelectSingleNode("accountName[@id='2']").'#text'
+        $ctx1 = (Get-AzStorageAccount -ResourceGroupName $resourceGroupName -Name $accountName).Context
+        $container = New-AzStorageContainer -Name $containerName -Context $ctx1
+        
+        $blob = Set-AzStorageBlobContent -Container $containerName -File $localSrcFile -Blob test1 -StandardBlobTier Cold -Properties @{"ContentType" = "image/jpeg"} -Metadata @{"tag1" = "value1"} -Context $ctx1
+        $blob.Name | Should -Be "test1"
+        $blob.AccessTier | Should -Be "Cold"
+        $blob.BlobProperties.ContentType | Should -Be "image/jpeg"
+
+        $blob = Set-AzStorageBlobContent -Container $containerName -File $localSrcFile2 -Blob test2 -StandardBlobTier Cold -Context $ctx1
+        $blob.Name | Should -Be "test2"
+        $blob.AccessTier | Should -Be "Cold"
+
+        $blob.BlobBaseClient.SetAccessTier("Cold")
+        $blob.AccessTier | Should -Be "Cold"
+        $blob.Name | Should -Be "test2"
+
+        $blob = Get-AzStorageBlob -Container $containerName -Blob test1 -Context $ctx1 
+        $blob.AccessTier | Should -Be "Cold"
+        $blob.Name | Should -Be "test1"
+        $blob.BlobProperties.ContentType | Should -Be "image/jpeg"
+
+        $destBlobName = "destblob1"
+        $copyblob = $blob | Copy-AzStorageBlob -DestContainer $containerName -DestBlob $destBlobName -Force
+        $copyblob.Name | Should -Be $destBlobName
+        $copyblob.AccessTier | Should -Be "Hot"
+        $copyBlob.BlobProperties.ContentType | Should -Be "image/jpeg"
+
+        $copyblob = $blob | Copy-AzStorageBlob -DestContainer $containerName -DestBlob $destBlobName -StandardBlobTier Cold -Force
+        $copyblob.Name | Should -Be $destBlobName
+        $copyblob.AccessTier | Should -Be "Cold"
+        $copyBlob.BlobProperties.ContentType | Should -Be "image/jpeg"
+
+        $largeBlob = Get-AzStorageBlob -Blob test2 -Container $containerName -Context $ctx1 
+        $copyblob = $largeblob | Copy-AzStorageBlob -DestContainer $containerName -DestBlob $destBlobName -StandardBlobTier Cold -Force
```

Comment:
We will run "Copy-AzStorageBlob" 3 times, but all with pieline.
Could we run at least once without pipeline, but all parameters input manually?

Reply:
Done